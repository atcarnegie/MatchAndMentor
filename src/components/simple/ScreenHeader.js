import React from 'react'
import { View, Text } from 'react-native'

function ScreenHeader ({title}) {
  return (
    <View style={styles.containerStyle}>
      <Text style={styles.textStyle}>{title}</Text>
    </View>
  )
}

const styles = {
  containerStyle: {
    margin: 20
  },
  textStyle: {
    fontSize: 24,
    color: 'black'
  }
}

export default ScreenHeader
