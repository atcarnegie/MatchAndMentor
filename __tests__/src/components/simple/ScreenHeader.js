/* eslint-env jest */
'use strict'
import 'react-native'
import React from 'react'
import ScreenHeader from '../../../../src/components/simple/ScreenHeader'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(
    <ScreenHeader />
  )
  expect(tree).toMatchSnapshot()
})
