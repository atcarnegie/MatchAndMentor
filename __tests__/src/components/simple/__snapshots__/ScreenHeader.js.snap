// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "margin": 20,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "black",
        "fontSize": 24,
      }
    }
  />
</View>
`;
